publish.css

```diff
@@ -25,28 +25,6 @@
     }    
 }
 
-
```

Comment:
Wait, so is this commit removing the publish css styling for `open in obsidian` links all together?

Reply:
it's only removing the pseudo-tooltip on hover? Or does it also remove something else I am not aware of? 🤔

Reply:
Like, only `:hover:before` is removed?

Reply:
Oh I see, we don't need that anymore?

Reply:
1. The tooltip says "you need the Hotkey Helper plugin for this"
2. This PR changes the URI, so we do not need the Hotkey helper plugin anymore.
3. Consequently, the tooltip that says "you need the Hotkey Helper plugin for this" is removed from the css.
😝